### PR TITLE
[13.0][ADD] survey_crm_generation: New module

### PR DESCRIPTION
**Pull request by chienandalu**
This module allows to generate leads/opportunities from surveys.

cc @Tecnativa TT40982

please review @victoralmau @pedrobaeza

### CODE REVIEW

File: survey_crm_generation/models/survey_user_input.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Copyright 2022 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class SurveyUserInput(models.Model):
+    _inherit = "survey.user_input"
+
+    opportunity_id = fields.Many2one(comodel_name="crm.lead")
```
Comment (victoralmau) [inline survey_crm_generation/models/survey_user_input.py:9]: Please add this field to form view.

**Review by victoralmau — APPROVED** [review]
Code and functional review OK.

File: survey_crm_generation/models/survey_user_input.py (hunk 1 above)
Comment (chienandalu) [reply survey_crm_generation/models/survey_user_input.py:9]: Done :+1:

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

Don't we have the SVG icon?

**Pull request merged**